src/option.cpp

```diff
@@ -898,9 +940,29 @@ void process_option_line(const std::string &config_line, const char *filename)
          }
       }
    }
+   else if (cmd == "using")
+   {
+      auto vargs = split_args(args[1], filename, is_varg_sep);
+      if (vargs.size() != 2)
+      {
+         OptionWarning w{ filename };
+         w("%s requires a version number in the form MAJOR.MINOR", cmd.c_str());
```

Comment:
Previously we had to release uncrustify-0.66.1 and uncrustify-0.68.1.
Why don't we use PATCH here as well?